assets/javascripts/discourse/models/chat-message.js

```diff
@@ -0,0 +1,4 @@
+import RestModel from "discourse/models/rest";
+
+const ChatMessage = RestModel.extend();
```

Comment:
Huge nitpick here but you needn't assign `ChatMessage` - you could `export default RestModel.extend()`